frontend/src/core/l10n/actions.js

```diff
@@ -31,9 +32,9 @@ export function request(): RequestAction {
  */
 export type ReceiveAction = {
     +type: typeof RECEIVE,
-    +bundles: Array<FluentBundle>,
+    +bundles: ReactLocalization,
```

Comment:
Nit: I'd call it `l10n` or `localization`, since `bundles` has a specific meaning inside of it. But it's not very important.